packages/amplify-graphql-index-transformer/src/__tests__/amplify-graphql-primary-key-transformer.test.ts

```diff
@@ -1006,6 +1006,37 @@ describe('RDS primary key transformer tests', () => {
     const getQuery = query.fields.find((f: any) => f.name.value === 'testGet');
     expect(getQuery).toBeDefined();
   });
+
+  it('id is treated as implicit primaryKey and necessary resolver resources are generated', () => {
+    const validSchema = `
+      type Song @model {
+        id: ID!
+        name: String!
+        genre: String!
+      }
+    `;
```

Comment:
Can we add a suite of tests that validates the entire app generated for this schema is the same as what is generated if there's an explicit `@primaryKey` on `id`? That'd give me more confidence that we're doing the right thing across all transformers in concert.